mappings/net/minecraft/client/util/NetworkUtils.mapping

```diff
@@ -1,5 +1,10 @@
 CLASS net/minecraft/class_3521 net/minecraft/client/util/NetworkUtils
-	FIELD field_15664 downloadExecutor Lcom/google/common/util/concurrent/ListeningExecutorService;
+	COMMENT A few client-side networking utilities.
+	COMMENT
+	COMMENT @implNote This is not marked as client-only because it's used by the
+	COMMENT {@code /publish} command, which is only available to integrated servers
+	COMMENT yet was retained by proguard.
+	FIELD field_15664 DOWNLOAD_EXECUTOR Lcom/google/common/util/concurrent/ListeningExecutorService;
 	FIELD field_15665 LOGGER Lorg/apache/logging/log4j/Logger;
-	METHOD method_15301 download (Ljava/io/File;Ljava/lang/String;Ljava/util/Map;ILnet/minecraft/class_3536;Ljava/net/Proxy;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_15301 downloadResourcePack (Ljava/io/File;Ljava/lang/String;Ljava/util/Map;ILnet/minecraft/class_3536;Ljava/net/Proxy;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
This rename reminds me, there was a bad server-resource-pack-related name in `ServerInfo`, do you think you could rename that too while you're at it?

Reply:
Do you mean that resource pack policy, like accept/reject/prompt enum and related stuff?

Reply:
Yeah that was it